ctapipe/instrument/camera/geometry.py

```diff
@@ -383,6 +383,36 @@ def _all_pixel_areas_equal(self):
         """
         return ~np.any(~np.isclose(self.pix_area.value, self.pix_area[0].value), axis=0)
 
+    @lazyproperty
+    def _pixel_rows(self):
```

Comment:
please make this public. It's useful and there is no reason why this should be hidden.

Reply:
Also, call it `pixel_row`, we don't use plural for arrays of stuff, compare pixel `x` or `area` or `diameter`